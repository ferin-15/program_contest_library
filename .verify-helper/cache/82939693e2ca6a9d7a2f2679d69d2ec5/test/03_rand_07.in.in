500 12
11 9 1 25 12 22 4 16 20 4 25 2 6 11 19 24 16 27 16 30 20 21 8 27 14 28 7 22 26 1 27 6 1 19 22 13 3 17 20 22 21 15 23 18 25 11 4 3 29 11 24 18 1 1 14 15 20 13 28 15 13 16 13 5 5 4 9 7 21 29 20 3 13 4 12 29 14 15 23 13 25 16 22 18 16 28 2 27 10 29 12 14 14 24 18 10 19 27 8 1 25 19 25 29 22 7 19 6 13 12 10 30 27 1 17 5 20 10 1 29 30 4 12 5 19 30 7 8 18 14 8 12 3 3 2 16 1 20 21 5 1 22 4 28 15 12 2 4 21 24 3 12 28 6 17 16 5 23 23 22 28 23 25 22 17 26 8 9 16 20 13 8 4 17 5 18 20 28 13 3 22 7 14 11 13 22 26 17 6 11 1 4 25 17 17 11 13 16 19 20 6 1 27 1 9 24 18 29 13 22 1 26 29 6 6 11 28 24 19 3 4 11 28 28 28 7 30 2 22 10 21 19 10 9 19 11 24 28 1 7 20 23 2 18 28 30 28 17 23 8 12 26 19 9 15 16 7 6 17 29 15 29 9 24 29 28 26 23 25 26 21 6 18 14 23 8 13 12 24 5 20 5 22 30 6 28 15 12 3 23 2 17 13 11 3 3 30 28 25 24 16 7 22 25 21 14 2 3 18 18 30 29 22 21 28 27 11 12 1 13 26 2 30 8 12 24 2 3 21 27 19 28 25 10 23 15 15 16 10 2 3 9 30 17 21 27 5 1 30 5 6 25 7 27 24 10 20 26 13 2 14 1 30 30 2 14 15 16 29 16 18 24 24 9 10 6 6 14 29 27 11 4 14 17 22 29 18 11 16 22 12 21 14 11 21 15 16 5 23 15 20 10 30 5 10 9 2 7 14 30 4 24 25 17 2 16 7 12 18 23 3 30 13 9 2 3 15 18 29 7 24 18 8 23 14 18 23 16 24 28 7 19 22 24 27 15 1 4 26 19 26 21 10 30 29 11 3 5 20 1 12 13 11 11 27 24 20 11 9 14 9 8 24 22 1 21 6 1 24 2 19 11 22 28 10 12 1 4 16 20 5 19 25 7 30 13 30 11 24 1
