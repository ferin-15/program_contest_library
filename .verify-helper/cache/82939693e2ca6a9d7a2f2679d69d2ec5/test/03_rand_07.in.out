1 1 1 2 2 2 2 2 2 2 2 2 6 8 8 7 7 7 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 3 3 3 3 3 3 3 1 1 1 1 1 1 1 1 1 1 1 1 1 4 4 4 4 4 4 3 3 3 3 3 3 3 3 3 3 3 3 4 4 12 2 2 2 2 2 2 2 2 2 2 2 2 8 1 1 1 1 1 1 1 1 1 1 1 1 6 6 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4 4 3 3 2 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 2 2 3 3 3 3 5 5 5 5 5 5 8 8 8 8 4 4 4 4 4 4 4 3 3 3 3 3 3 3 3 3 3 3 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 3 3 3 3 2 2 2 2 2 2 2 2 2 2 2 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 8 8 7 6 6 6 6 6 6 6 6 6 6 6 6 9 9 6 6 6 6 6 6 6 6 5 5 5 5 5 5 5 5 5 3 3 2 2 2 2 2 2 2 2 2 2 2 2 3 3 3 3 2 2 2 2 2 2 2 2 2 2 2 2 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 2 2 2 2 2 3 2 2 2 2 2 2 2 2 1 1 1 1 1 1 1 1 1 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 1 1 2 2 2 6 6 6 6 6 4 4 4 4 4 4 4 4 4 4 4 4 11 11 11 11 5 5 5 5 5 5 5 5 5 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 3 7 7 7 7 7 7 7 7 7 1 1 1 1 1 1 1 1 1 1 1 1 3 1 1 1 1 1 1 1 1 1 1 1 1 8 8 8 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4 1
